themes/bootstrap3/theme.config.php

```diff
@@ -64,5 +63,172 @@
             'layoutClass' => 'VuFind\View\Helper\Bootstrap3\LayoutClass',
             'search' => 'VuFind\View\Helper\Bootstrap3\Search'
         ]
-    ]
+    ],
+    'icons' => [
+        'defaultSet' => 'FontAwesome',
+        'sets' => [
+            /**
+             * Define icon sets here.
+             *
+             * All sets need:
+             * - 'template': which template the icon renders with
+             * - 'src': the location of the relevant resource (font, css, images)
+             * - 'prefix': prefix to place before each icon name for convenience
+             *             (ie. fa fa- for FontAwesome, default "")
+             */
+            'FontAwesome' => [
+                'template' => 'font',
+                'prefix' => 'fa fa-',
+                'src' => 'vendor/font-awesome.min.css',
+            ],
+            'Collapse' => [
+                'template' => 'collapse',
+            ],
+            /* For an example of an images set, see Bootprint's theme.config.php. */
+        ],
+        'aliases' => [
+            /**
+             * Icons can be assigned or overriden here
+             *
+             * Format: 'icon' => [set:]icon
+             * Icons assigned without set will use the defaultSet.
+             *
+             * All of the items below have been specified with FontAwesome to allow
+             * for a strong inheritance safety net but this is not required.
+             */
+            // UI
+            'barcode' => 'FontAwesome:barcode',
```

Comment:
Could the list be alphabetically sorted?

Reply:
@crhallberg just a reminder of this conversation: I think it may be useful to alphabetize the list now that we're prefixing things in meaningful ways.